cpp/harvest/marc_issn_lookup.cc

```diff
@@ -57,11 +58,14 @@ struct SubFieldInfo {
     std::string t_;
```

Comment:
As discussed, i think it would be better to rename SubFieldInfo to CacheEntry and use actual property names instead of the short values in this struct.

Reply:
That looks a lot better, thanks!